apps/webpack.js

```diff
@@ -30,6 +30,7 @@ var baseConfig = {
       '@cdo/apps': path.resolve(__dirname, 'src'),
       '@cdo/static': path.resolve(__dirname, 'static'),
       repl: path.resolve(__dirname, 'src/noop'),
+      http: 'stream-http',
```

Comment:
I believe we have some weirdness around `stream-http`. It's probably worth documenting that somewhere (maybe here?)